Tests/ClientTests/TelemetryWrapperTests.swift

```diff
@@ -165,31 +165,6 @@ class TelemetryWrapperTests: XCTestCase {
     }
 
     // MARK: - Onboarding
-
-    func test_onboardingCardViewWithExtras_GleanIsCalled() {
```

Comment:
For context on why these were removed, they're technically the same things tested in the `OnboardingTelemetryUtilityTests`